docs/deploy.rst

```diff
@@ -129,7 +150,8 @@ deployment, exodus-lambda-deploy.yaml, can be run by the `AWS CLI`_ command
         --template-file configuration/exodus-lambda-pkg.yaml \
         --stack-name ... \
         --capabilities ... \
-        --parameter-overrides env=$ENV_TYPE oai=... project=$PROJECT
+        --parameter-overrides env=$ENV_TYPE project=$PROJECT \
+            oai=... lambdafunctionrole=...
```

Comment:
> [...] I'll add some suggestions to illustrate what I have in mind.

This one was okay how you originally had it.
I missed the environment variables and script when I suggested removing the vars from the aws command.
```
 $ export $PROJECT=...
 $ export $ENV_TYPE=...
 $ ./scripts/build-package
 $ aws cloudformation deploy \
        --template-file configuration/exodus-lambda-pkg.yaml \
        --stack-name ... \
        --capabilities ... \
        --parameter-overrides env=$ENV_TYPE oai=... \
            project=$PROJECT lambdafunctionrole=...
```
The environment variables makes sense to use here in the aws command because they're defined in the example already.